server/container/util/src/main/java/org/apache/james/util/FluentFutureStream.java

```diff
@@ -167,6 +195,14 @@ private FluentFutureStream(CompletableFuture<Stream<T>> completableFuture) {
         return CompletableFutureUtil.reduce(combiner, completableFuture, emptyAccumulator);
     }
 
+    /**
+     * sort all elements of the stream by the provided {@code Comparator}.
+     */
+    public FluentFutureStream<T> sorted(Comparator<T> comparator) {
```

Comment:
I would not add this function to this already too big helper as it's probably rarely usefull

Reply:
It just takes few lines of code! If No, I have to sort elements from`FluentFutureStream` in the ugly way. (access the `CompletableFuture<Stream>` then `CompletableFuture.apply()` then `Stream.sorted()`)